pkg/cortex/modules.go

```diff
@@ -6,6 +6,10 @@ import (
 	"fmt"
 	"net/http"
 
+	"github.com/thanos-io/objstore"
+
+	"github.com/cortexproject/cortex/pkg/storage/bucket"
+
```

Comment:
Nit. It would be great if you could group the imports. Like if importing cortex packages, it usually goes to the last section.

Reply:
Oh thanks for the suggestion, I think I hadn't configured my `goimports` for that yet.